torch/_inductor/fx_passes/quantization.py

```diff
@@ -142,3 +164,196 @@ def qconv(match: Match, *args, **kwargs):
 
 def register_quantization_lowerings():
     _register_quantized_conv_lowering(quantize_conv_output_pattern)
+
+
+def _is_valid_dequant_conv2d_pattern(match):
```

Comment:
could you add some comments for this function? what is this doing?

Reply:
Thanks and comments added.